### ISSUE TITLE
Can't connect Gmail

### ISSUE THREAD

**Issue by Bollaoros** [getstation/desktop-app#50]
It gives me a client ID error when I try and integrate my Gmail

**Comment by Steve667** [comment]
Having the same error, actually. 
```
Error 400: invalid_request
Missing required parameter: client_id
```
OS: macOS Big Sur 11.4

**Comment by LoKy-dev** [comment]
Duplicae of #2

**Comment by LoKy-dev** [comment]
Duplicate of #2

**Comment by BayAreaData** [comment]
I third this. Gmail won't authorize, both Google accounts and Google-managed company accounts.

**Comment by apostolisgan7** [comment]
Hey there! me too, i've got the same problem!

**Comment by lotusp** [comment]
I got the same error:

Error 400： invalid_request
Missing required parameter: client_id

macOS Big Sur 11.4

**Comment by FiciencyAg** [comment]
Same here

**Comment by davidbenucci** [comment]
Me too

**Comment by SuperSonicSites** [comment]
Same here

**Comment by raejbrown** [comment]
I am having the same issue. Anything resolved yet?

Error 400: invalid_request
Missing required parameter: client_id

**Comment by Aynril** [comment]
I have the same issue, any fixes available yet?

**Comment by wjani** [comment]
Hello!
This is my problem too.
Is there a solution to your gmail account error?

**Comment by Techflow-ai** [comment]
I have the issue too.

**Comment by Serubin** [comment]
This needs to be solved by Station. They need to include their developer client id, I'm fairly certainly.

**Comment by Akansh0581** [comment]
I have got the same issue
@getstation, please include the client Id as soon as possible
Otherwise the software would be useless

**Comment by robm87** [comment]
Same problem here, I updated Station to newest version (lost all my old accounts :( ), now I want to configure all my accounts again. multiple problems; Gmail > invalid request, Whatsap > no QR code,

**Comment by quillevere** [comment]
Same here. It's starting to become a real problem and not having a fix or even updates after 10 days is a bit frustrating...

**Comment by smallz207** [comment]
SAME! It was working for me before... but now it's not and I'm getting the same error!

**Comment by convertl** [comment]
Same problem here for a new Station installation on Windows

**Comment by Orbital101Studio** [comment]
Same problem here. All google Apps dont work with business and personal accounts providing the same error message : 

Gmail auth error:
Error 400: invalid_request
Missing required parameter: client_id

**Comment by joeknouse** [comment]
Yup. Same problem. Mac user.

**Comment by jkmgeo** [comment]
Not tied to Station (or Adalo) in any way, but some quick searching after encountering the same problem suggests that this may be related to needing a Google developer client ID like mentioned in this link, especially steps 3 & 4: https://help.adalo.com/component-basics/marketplace-components/sign-in-with-google

I personally lack the know-how to update the Station app to rectify this, but maybe this points some dev in the right direction to get this issue fixed for all of us? @getstation 

Pinging issues #2 #28 #29 #32 #40 #59 #60, as well.

**Comment by ameliasellsmaine** [comment]
curious if we are ever going to get a fix for this....

**Comment by GitToby** [comment]
looks like #70 and #68 too now

**Comment by viktor44** [comment]
Fixed in [2.1.0](https://github.com/getstation/desktop-app/releases/tag/v2.1.0)
